bin/stop-shuffle-server.sh

```diff
@@ -20,10 +20,8 @@
 set -o pipefail
 set -o nounset   # exit the script if you try to use an uninitialised variable
 set -o errexit   # exit the script if any statement returns a non-true return value
-set -e
```

Comment:
unify style

`set -e` same as `set -o errexit`: https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html